test/form-helper.test.js

```diff
@@ -191,7 +191,11 @@ describe('form-helper functions', () => {
                     cspNonce
                 })
                 .replace(/\s+/g, '');
-
+            // ignore the version tag in bundle - empty version included in pattern as it doesn't generate properly when the test is run manually
+            actual = actual.replace(
+                /\/dist\/js\/bundle\.js\?(\d+\.\d+\.\d+)*/,
+                '/dist/js/bundle.js'
+            );
```

Comment:
Can you not just update the [fixture](https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/df0463380e0cc8d0fc02d09acd625639d82804ba/test/test-fixtures/transformations/resolved%20html/p-applicant-british-citizen-or-eu-national.js#L549) to have some arbitrary `?1.2.3` version on the end of it and then have that as the value of the test `process.env.npm_package_version` in the [jest.setup.js](https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/master/jest.setup.js) file?

This would avoid "hacky" string manipulation in a test